src/Dependencies/PooledObjects/Microsoft.CodeAnalysis.PooledObjects.shproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
-    <ProjectGuid>cd77a8cc-2885-47a7-b99c-fbf13353400b</ProjectGuid>
+    <ProjectGuid>c1930979-c824-496b-a630-70f5369a636f</ProjectGuid>
```

Comment:
Looks like we have a misauthoring in our solution.  This is VS just trying to keep things in sync.  I've got htis change in all my PRs. Once it gets fixed up, it'll dissapear from all PRs.